etc/org.opencastproject.composer.impl.ComposerServiceImpl.cfg

```diff
@@ -0,0 +1,18 @@
+# Job load for process-smiltrack and multiencode is calculated based on the encoding profiles used.
+# Their job loads are summed up and multiplied by a factor.
+# The factor represents the adjustment that should be made assuming each profile job load was specified
+# based on it running with one input -> one output so normally will be a number 0 < n <= 1
+# Their job loads maxes out at job.load.max.multiple.profiles.
+# Default: 0.8
+#job.load.max.multiple.profiles=0.8
+
+# The process-smiltrack operation stitches all the editor segments and transcode with multiple
+# encoding profiles concurrently.
+# Default: 0.5
+#job.load.factor.process.smil=0.5
+
+# Transition duration in seconds between each edited section when using process-smiltrack operation
+# If it is set to 0, there is no transition between each segment
+# By default, it is a 2 second fade-to-black and fade-from-black and audio fading between each segment
+
+org.composer.process_smil.edit.transition.duration = 2
```

Comment:
Please adapt to the same style as the configuration properties above, i.e. don't explicitly set the default value, same style of documentation of default value ("# Default: 2)

Reply:
Hi @phoebem 

I've found the following problem while testing:

Step 1: A video is cutted...

![screen shot 2018-06-13 at 14 53 45](https://user-images.githubusercontent.com/1590263/41353574-e92dd3e0-6f1c-11e8-81ad-b40e7bef0c3b.png)

Step 2: To verify that the standard workflow does not break things, it is started (Video Editor->Publish). After the workflow has processed the video, things look like:

![screen shot 2018-06-13 at 15 09 51](https://user-images.githubusercontent.com/1590263/41353611-04dcb002-6f1d-11e8-96fe-8ecb4c2116e0.png)

Problem: If the new process-smil workflow is started from the video editor, processing more or less works (the result is cutted, but the Engage player still shows the full length of the video), but also, when opening the video editor again, things look like:

![screen shot 2018-06-13 at 15 16 12](https://user-images.githubusercontent.com/1590263/41353667-2e777b0e-6f1d-11e8-9086-8092a31c7ae4.png)


I have verified that the segments in the timeline don't get messed up in opencast/develop or in this branch when not using the new process-smil workflow.

Can you please have a look at this issue?